language/extensions/move-table-extension/src/lib.rs

```diff
@@ -352,7 +374,9 @@ fn native_new_table_handle(
     Digest::update(&mut digest, table_data.new_tables.len().to_be_bytes());
     let bytes: [u8; 16] = digest.finalize()[0..16].try_into().unwrap();
     let id = u128::from_be_bytes(bytes);
-    assert!(table_data.new_tables.insert(TableHandle(id)));
+    let key_type = get_type_layout(context, &ty_args[0])?;
+    let value_type = get_type_layout(context, &ty_args[1])?;
```

Comment:
I'm under the impression `ty_args[2]` is the value type, but I don't remember why.

Reply:
I'm stupid. ignore me.